pl/deploy/signup_pythonanywhere.md

```diff
@@ -6,10 +6,14 @@ Utwórz konto dla "Beginner" na PythonAnywhere (darmowy poziom jest dobry, nie p
 
 ![Strona rejestracji PythonAnywhere z przyciskiem, aby utworzyć bezpłatne konto "Beginner"](../deploy/images/pythonanywhere_beginner_account_button.png)
 
-> ** Uwaga ** Wybierając tutaj swoją nazwę użytkownika, należy pamiętać, że adres URL bloga będzie miał postać `yourusername.pythonanywhere.com`, więc wybierz swój własny nick lub nazwę tego, o czym będzie blog.
+> ** Uwaga ** Wybierając tutaj swoją nazwę użytkownika, należy pamiętać, że adres URL bloga będzie miał postać `twojanazwa.pythonanywhere.com`, więc wybierz swój własny nick lub nazwę tego, o czym będzie blog. Pamiętaj, aby zapamiętać hasło (dodaj je do menedżera haseł, jeśli go używasz).
```

Comment:
Everywhere else you use `<your-pythonanywhere-domain` instead of `twojanazwa` in `twojanazwa.pythonanywhere.com`. Is it changed here on purpose?